### PR TITLE
doc: Add 2.1.3 to main

### PR DESCRIPTION
**Pull request by peknis**
Adding the 2.1.3 docs to main branch.

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

**Review by rlubos — APPROVED** [review]

**Review by martintv — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -949,14 +950,14 @@ KRKNWK-15749: Invalid ZAP Tool revision used
...
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0
+.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0
 
 KRKNWK-14473: Unreliable communication with the window covering sample
   The :ref:`window covering sample <matter_window_covering_sample>` might rarely become unresponsive for a couple of seconds after commissioning to the Matter network.
 
   **Workaround:** Switch from SSED to SED role.
 
-.. rst-class:: v2-2-0 v2-1-2 v2-1-1 v2-1-0
+.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0
```
Comment (umapraseeda) [inline doc/nrf/known_issues.rst:960]: Line 988: 2-1-3 missing?

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -1893,7 +1902,7 @@ CIA-248: Samples with default SPM config fails to build for ``thingy91_nrf9160_n
 nRF Profiler
 ============
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```
Comment (umapraseeda) [inline doc/nrf/known_issues.rst:1905]: NCSIDB-114 missing 2-1-3? (line 1873)

Comment (peknis) [reply doc/nrf/known_issues.rst:1905]: Fixed

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (peknis) [reply doc/nrf/known_issues.rst:960]: Fixed

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -189,22 +190,22 @@ NCSDK-15471: Compilation with SUPL client library fails when TF-M is enabled
...
-.. rst-class:: v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
 
 NCSDK-10106: Elevated current consumption when using applications without :ref:`nrfxlib:nrf_modem` on nRF9160
   When running applications that do not enable :ref:`nrfxlib:nrf_modem` on nRF9160 with build code B1A, current consumption will stay at 3 mA when in sleep.
 
   **Workaround:** Enable :ref:`nrfxlib:nrf_modem`.
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
+.. rst-class:: v2-1-2 v2-1-3 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
```
Comment (umapraseeda) [inline doc/nrf/known_issues.rst:208]:
```suggestion
.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
```

**Review by umapraseeda — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]
LGTM :+1:

**Review by czeslawmakarski — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**